WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -505,5 +555,14 @@ private extension StoreStatsAndTopPerformersViewController {
 
     enum Constants {
         static let backgroundColor: UIColor = .systemBackground
+        static let customRangeTabAcessibilityIdentifier: String = "period-data-custom-tab"
+    }
+
+    enum Localization {
+        static let customRangeTabTitle = NSLocalizedString(
+            "storeStatsAndTopPerformersViewController.customRangeTabTitle",
+            value: "Custom Range",
+            comment: "Title of the Custom Range tab"
+        )
```

Comment:
These can be removed, as explained in previous comments.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/bcd123be653bc29aa63bbe9507d339f8eea68a90